openmoc/plotter.py

```diff
@@ -225,12 +229,22 @@ def plot_materials(geometry, gridsize=250, xlim=None, ylim=None):
 
   if not is_integer(gridsize):
     py_printf('ERROR', 'Unable to plot the Materials since ' + \
-              'since the gridsize %d is not an integer', gridsize)
+              'the gridsize %d is not an integer', gridsize)
 
   if gridsize <= 0:
     py_printf('ERROR', 'Unable to plot the Materials ' + \
               'with a negative gridsize (%d)', gridsize)
-
+    
+  if z_level is None:
```

Comment:
Sometimes I've noticed differences with "is" and "==", not sure if it makes a difference here